docs/fides/docs/getting-started/database_connectors.md

```diff
@@ -424,6 +424,10 @@ To view a list of all available connection types, visit `GET /api/v1/connection_
             "identifier": "datadog",
             "type": "saas"
         },
+        {
+            "identifier": "fullstory",
```

Comment:
It doesn't hurt to include this but we're not keeping this up-to-date since it's just an example of the API response